huaweicloud/services/rds/resource_huaweicloud_rds_lts_log.go

```diff
@@ -0,0 +1,267 @@
+				Description: `Specifies the ID of the RDS instance.`,
+			},
+			"engine": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the engine of the RDS instance.`,
+			},
+			"log_type": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the type of the LTS log.`,
+			},
+			"lts_group_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the ID of the LTS log group.`,
+			},
+			"lts_stream_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the ID of the LTS log stream.`,
+			},
+		},
+	}
+}
+
+func buildCreateRdsLtsLogBodyParams(d *schema.ResourceData) map[string]interface{} {
+	ltsConfigs := map[string]interface{}{
+		"log_configs": []map[string]interface{}{
+			{
+				"instance_id":   d.Get("instance_id"),
+				"log_type":      d.Get("log_type"),
+				"lts_group_id":  d.Get("lts_group_id"),
+				"lts_stream_id": d.Get("lts_stream_id"),
+			},
+		},
+	}
+	return ltsConfigs
+}
+
+func resourceRdsLtsLogCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+	engine := d.Get("engine").(string)
+	instanceID := d.Get("instance_id").(string)
+	logType := d.Get("log_type").(string)
+
+	var (
+		createRdsLtsLogHttpUrl = "v3/{project_id}/{engine}/instances/logs/lts-configs"
+		createRdsLtsLogProduct = "rds"
+	)
+	createRdsLtsLogClient, err := cfg.NewServiceClient(createRdsLtsLogProduct, region)
+	if err != nil {
+		return diag.Errorf("error creating RDS client: %s", err)
+	}
+
+	createRdsLtsLogPath := createRdsLtsLogClient.Endpoint + createRdsLtsLogHttpUrl
+	createRdsLtsLogPath = strings.ReplaceAll(createRdsLtsLogPath, "{project_id}", createRdsLtsLogClient.ProjectID)
+	createRdsLtsLogPath = strings.ReplaceAll(createRdsLtsLogPath, "{engine}", engine)
+
+	createRdsLtsLogOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		OkCodes: []int{
+			204,
+		},
+	}
+	createRdsLtsLogOpt.JSONBody = utils.RemoveNil(buildCreateRdsLtsLogBodyParams(d))
+
+	retryFunc := func() (interface{}, bool, error) {
+		_, err = createRdsLtsLogClient.Request("POST", createRdsLtsLogPath, &createRdsLtsLogOpt)
+		retry, err := handleMultiOperationsError(err)
+		return nil, retry, err
+	}
+	_, err = common.RetryContextWithWaitForState(&common.RetryContextWithWaitForStateParam{
+		Ctx:          ctx,
+		RetryFunc:    retryFunc,
+		WaitFunc:     rdsInstanceStateRefreshFunc(createRdsLtsLogClient, instanceID),
+		WaitTarget:   []string{"ACTIVE"},
+		Timeout:      d.Timeout(schema.TimeoutCreate),
+		DelayTimeout: 10 * time.Second,
+		PollInterval: 10 * time.Second,
+	})
+	if err != nil {
+		return diag.Errorf("error associating RDS with LTS log: %s", err)
+	}
+
+	d.SetId(fmt.Sprintf("%s/%s/%s", engine, instanceID, logType))
```

Comment:
```suggestion
	d.SetId(fmt.Sprintf("%s/%s",  instanceID, logType))
```
the instance engine can be get by the instance id

Reply:
done